### DISCUSSION TITLE
generator：请问是否允许不读取parser输出，在parser阶段直接生成IR？

### DISCUSSION THREAD

**Discussion by tiger9tu** [arcsysu/SYsU-lang#97] [General]
对于在实验二使用了AST的同学来说，实验三其实可以可以利用parser阶段构建的AST直接生成IR。请问是否允许自行修改测试代码，从而不接收parser输出而直接生成IR？

**Comment by Yorkking** [comment]
不行，因为不方便评测。但是你可以在完成了所有实验后，把parser和generator结合在一起，实现一个编译器。
